modules/ts/misc/run.py

```diff
@@ -879,7 +916,7 @@ def getRunArgs(args):
     run_args = getRunArgs(args[1:] or ['.'])
```

Comment:
@alalek However, it is not obvious and such help message may be confusing for user who is not familiar with run.py code when he is starting script from location other then binary directory